src/ansi-c/expr2c.cpp

```diff
@@ -738,19 +738,18 @@ std::string expr2ct::convert_struct_type(
   {
     dest+=" {";
 
-    for(struct_typet::componentst::const_iterator
-        it=struct_type.components().begin();
-        it!=struct_type.components().end();
-        it++)
+    for(const struct_union_typet::componentt &component :
+      struct_type.components())
     {
       // Skip padding parameters unless we including them
-      if(it->get_is_padding() && !inc_padding_parameters)
+      if(component.get_is_padding() && !inc_padding_parameters)
       {
         continue;
       }
 
       dest+= ' ';
-      dest+=convert_rec(it->type(), c_qualifierst(), id2string(it->get_name()));
+      dest+=convert_rec(component.type(), c_qualifierst(),
+                         id2string(component.get_name()));
```

Comment:
Nit picking: use the same indent on the second line as the character after the "(", or even better put each of `component.type()`, `c_qualifierst()`, and `id2string(component.get_name())` on a new line.